javascript/ql/lib/semmle/javascript/internal/ConceptsShared.qll

```diff
@@ -67,12 +70,15 @@ module Cryptography {
       /** Gets an input the algorithm is used on, for example the plain text input to be encrypted. */
       abstract DataFlow::Node getAnInput();
 
+      /** DEPRECATED. This predicate has been renamed to `getAnInput`. */
+      deprecated final DataFlow::Node getInput() { result = this.getAnInput() }
```

Comment:
Same as above.

Reply:
`CryptographicOperation::Range#getInput()` has been removed - I don't think there's a need to include this as the `::Range` class is new for JS.